### ISSUE TITLE
Error should be generated if you attempt to bind to an unspecialized generic type

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#1238]
This should generate an error:

```python
T = TypeVar("T")

class Foo(Generic[T]):
     pass

C = TypeVar("C", bound=Foo)
```

**Comment by erictraut** [comment]
This will be included in the next release.

**Comment by erictraut** [comment]
This is now fixed in Pyright 1.1.92, which I just published. It will also be addressed in next week's Pylance release.
